client/lib/string/index.js

```diff
@@ -17,6 +8,16 @@
  * @returns
  */
 export function areEqualIgnoringWhitespaceAndCase( a, b ) {
+	// Are they equal without any manipulation?
+	if ( a === b ) {
+		return true;
+	}
+
+	// If we don't have strings for this part, bail out
+	if ( a === null || b === null ) {
```

Comment:
these should probably just be falsey checks? otherwise `undefined` can slip through and blow up the code below.

Reply:
Good point. We don't want any more 💥 .